### ISSUE TITLE
Add configuration options for hyperparameters

### ISSUE THREAD

**Issue by rarick** [rarick/football#18]
Learning rate, batch size, etc
